content/sensu-go/5.14/getting-started/learn-sensu.md

```diff
@@ -116,29 +113,27 @@ sensuctl event list
 sensu-go-sandbox   keepalive   Keepalive last sent from sensu-go-sandbox at 2019-01-24 21:29:06 +0000 UTC        0   false      2019-01-24 21:29:06 +0000 UTC 
 {{< /highlight >}}
 
-The sensu-go-sandbox keepalive event has status 0, meaning the agent is in an OK state and able to communicate with the Sensu backend.
+The sensu-go-sandbox keepalive event has status 0, which means the agent is in an OK state and is able to communicate with the Sensu backend.
 
-We can also see the event and the entity in the [Sensu dashboard](http://localhost:3002).
-Log in to the dashboard as the default admin user: username `admin` and password `P@ssw0rd!`.
+You can also see the event and the entity in the [Sensu dashboard](http://localhost:3002). Log in to the dashboard with the default admin credentials: username `admin` and password `P@ssw0rd!`.
 
 ## Lesson \#2: Pipe keepalive events into Slack
 
-Now that we know the sandbox is working properly, let's get to the fun stuff: creating a workflow.
-In this lesson, we'll create a workflow that sends keepalive alerts to Slack.
-(If you'd rather not create a Slack account, you can skip ahead to [lesson 3](#lesson-3-automate-event-production-with-the-sensu-agent).)
+Now that you know the sandbox is working properly, let's get to the fun stuff: creating a workflow. In this lesson, you'll create a workflow that sends keepalive alerts to Slack.
+
+_**NOTE**: If you'd rather not create a Slack account, you skip ahead to [Lesson \#3](#lesson-3-automate-event-production-with-the-sensu-agent)._
```

Comment:
I'd keep "can" here -- "you can skip ahead" sounds a little more friendly than "you skip ahead"